modules/nodes-cluster-worker-latency-profiles-about.adoc

```diff
@@ -8,27 +8,27 @@
 [id="nodes-cluster-worker-latency-profiles-about_{context}"]
 = Understanding worker latency profiles
 
-Worker latency profiles are multiple sets of carefully-tuned values for the `node-status-update-frequency`, `node-monitor-grace-period`, `default-not-ready-toleration-seconds` and `default-unreachable-toleration-seconds` parameters. These parameters let you control the reaction of the cluster to latency issues without needing to determine the best values manually.
+Worker latency profiles are four different categories of carefully-tuned parameters. The four parameters which implement these values are `node-status-update-frequency`, `node-monitor-grace-period`, `default-not-ready-toleration-seconds` and `default-unreachable-toleration-seconds`. These parameters can each take values which let you control the reaction of the cluster to latency issues without needing to determine the best values using manual methods.
```

Comment:
Avoid "take" and "let". 
.... These parameters can use values which allows you to control...."